<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nunito SemiBold</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="pBtnFileDownload">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>280</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito SemiBold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lEdtFileDownloadLink">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>561</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito SemiBold</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Enter download link here</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lEdtFileDownloadLocation">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito SemiBold</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Use browse button to enter save location</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pBtnFileSaveLocation">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>130</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="pBarFileDownload">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>601</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>851</width>
        <height>401</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>One Video</string>
       </attribute>
       <widget class="QLineEdit" name="lEdtVideoUrl">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>481</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nunito SemiBold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Enter video URL</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lEdtVideoDownloadLocation">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>100</y>
          <width>481</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nunito SemiBold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Use browse button to enter save location</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pBtnVideoSaveLocation">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>100</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>B</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="pBarVideoDownload">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>220</y>
          <width>601</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pBtnVideoDownload">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>280</y>
          <width>211</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nunito SemiBold</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Download</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nunito SemiBold</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Video Quality</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cBoxVideoQuality">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>170</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pBtnVideoData">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>30</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Full Playlist</string>
       </attribute>
       <widget class="QLineEdit" name="lEdtPlaylistDownloadLocation">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>100</y>
          <width>481</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nunito SemiBold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Use browse button to enter save location</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>170</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nunito SemiBold</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Video Quality</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lEdtPlaylistUrl">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>481</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nunito SemiBold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Enter Playlist URL</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cBoxPlaylistVideoQuality">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>170</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>high</string>
         </property>
        </item>
       </widget>
       <widget class="QProgressBar" name="pBarPlaylistDownload">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>260</y>
          <width>601</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>30</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>G</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pBtnPlaylistSaveLocation">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>100</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>B</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pBtnPlaylistDownload">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>310</y>
          <width>211</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nunito SemiBold</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Download</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>210</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nunito SemiBold</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Current Video : </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>210</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nunito SemiBold</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Total Videos in Playlist : </string>
        </property>
       </widget>
       <widget class="QLabel" name="lblCurrentVideoNumber">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>210</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lblTotalVideoCount">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>210</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
